sentences/nl/climate_HassClimateGetTemperature.yaml

```
@@ -4,4 +4,4 @@ intents:
     data:
     - sentences:
         - "hoe warm is het"
-        - "hoe warm is het in <area>"
+        - "hoe warm is het [in] <area>"
```

We matchen nou niet de zin "Hoe warm is de woonkamer".

`<area>` heeft ook al een `de|het`, dus zou dit logischer zijn?

```suggestion
        - "hoe warm is [het in] <area>"
```